packages/theme/src/__tests__/__snapshots__/theme.test.tsx.snap

```diff
@@ -61,6 +61,23 @@ Object {
     "colorBorderWarning": "#D97706",
     "colorBorderWeaker": "#CBD5E1",
     "colorBorderWeakest": "#E2E8F0",
+    "colorCustomButtonBackgroundPrimary": "#102EE9",
```

Comment:
Do we need the custom prefix? If not, I would prefer to remove it.
We will use `colorCustomButtonBackgroundPrimary` for all primary buttons, right? So, maybe we can call it just  `colorButtonBackgroundPrimary`.

Reply:
I don't know if we have exceptions, like using the `colorCustomSecondary` and `colorSecondary`. If yes, maybe we can think about something for the `colorCustomSecondary`.